.github/workflows/rust.yml

```diff
@@ -27,6 +27,6 @@ jobs:
     - name: Build
       run: cargo build --verbose
     - name: Run tests 
-      run: cargo test --verbose
+      run: cargo test -- copy_file --nocapture
```

Comment:
Why only copy_file?

Reply:
I'm just testing why the test fails on the CI and nowhere else :-)

Reply:
The issue is fixed, so this change was removed